ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/list/list.html

```diff
@@ -0,0 +1,36 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2016 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+  <ol data-sly-use.list="com.adobe.cq.wcm.core.components.models.List"
```

Comment:
@Rajamouli-hcl  is this correct UI for list?

![image](https://user-images.githubusercontent.com/63295621/85294085-d72a9e80-b4bb-11ea-8617-fbb30dd2e44a.png)

Reply:
No we need select variations for  Orderd and Unorderd List from Stylesystem        
![image](https://user-images.githubusercontent.com/64774076/85312348-575cfe00-b4d4-11ea-9b8f-aa1a9c0f6c3d.png)
![image](https://user-images.githubusercontent.com/64774076/85312827-0568a800-b4d5-11ea-9fd5-1339e895e8ce.png)

Reply:
@Rajamouli-hcl  you missed the policies. Pl push the policies for style system.

Reply:
added policies as per review comment